README.md

```diff
@@ -0,0 +1,129 @@
+# sphinx-heroku-buildpack
+A Heroku buildpack to generate launch a sphinx documentation based on the same repository.
+
+For more information read about [Heroku Buildpacks](https://devcenter.heroku.com/articles/buildpacks)
+
+## Setting up sphinx-heroku-buildpack
+
+* Create heroku app to host sphinx documentation.
+
+```shell
+$heroku create myapp-docs
+```
+
+The order of buildpacks on your application is important!
+
+1.  First setup heroku-buildpack-graphviz
+```shell
+$heroku buildpacks:set https://github.com/weibeld/heroku-buildpack-graphviz.git
+```
+
+2.  Setup sphinx-heroku-buildpack
+```shell
+$heroku buildpacks:set https://github.com/Thermondo/sphinx-heroku-buildpack.git
+```
+
+3. Finally setup heroku's official python buildpack
+```shell
+$heroku buildpacks:set heroku/python
+```
+
+When listing the buildpacks on our application, we get this result:
+
+```shell
+$heroku buildpacks --app myapp-docs
+=== myapp-docs Buildpack URLs
+1. https://github.com/weibeld/heroku-buildpack-graphviz.git
+2. https://github.com/Thermondo/sphinx-heroku-buildpack.git
+3. heroku/python
+```
+
+## Why?
+
+To reduce deployment time by serving sphinx documentation on a separate Heroku app but still using the same source code for your application.
```

Comment:
You might want to get into Heroku auto deploy for a minute. Because that is what we do here. It doesn't even work without auto deploy, well you'd have to do it manually.